contracts/modules/cover/Cover.sol

```diff
@@ -735,4 +735,12 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard, Mu
     internalContracts[uint(ID.MR)] = master.getLatestAddress("MR");
     internalContracts[uint(ID.CP)] = master.getLatestAddress("CP");
   }
+
+  function changeNFTDescriptor(address _coverNFTDescriptor) external onlyInternal {
+    coverNFT.changeNFTDescriptor(_coverNFTDescriptor);
+  }
```

Comment:
As discussed:
- rename to `changeCoverNftDescriptor`
- add a similar function for the staking nft descriptor
- change modifier to allow ab member to trigger this instead of gov vote